src/EncompassRest/Loans/Enums/FreddieMacArmIndexType.cs

```diff
@@ -0,0 +1,22 @@
+using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.Enums
+{
+    public enum FreddieMacArmIndexType
+    {
+        [EnumMember(Value = "01")]
```

Comment:
numeric values to match value (eg. 02 = 2, 99 = 99)

Reply:
In case you didn't see I've done as you suggested.